docs/lib/datastore/fragments/js/sync/50-selectiveSync.md

```diff
@@ -108,7 +123,7 @@ DataStore.configure({
 ```
 
 ### Advanced use case - Query instead of Scan
-You can utilize Selective Sync in order to have your base sync retrieve items from DynamoDB by performing a [query operation](https://docs.aws.amazon.com/amazondynamodb/latest/APIReference/API_Query.html) against a GSI. By default, the base sync will perform a scan.
+You can configure selective sync to retrieve items from DynamoDB with a [query operation](https://docs.aws.amazon.com/amazondynamodb/latest/APIReference/API_Query.html) against a GSI. By default, the base sync will perform a scan. Learn about creating GSIs with the `@key` directive [here](https://docs.amplify.aws/cli/graphql-transformer/key).
```

Comment:
reading this again, we should add one more line on why query & scan operations are different. Add it right after: "perform a scan. Query operations enable..."